src/main/java/com/example/Huiswerkles10backend/dtos/input/TelevisionInputDto.java

```diff
@@ -0,0 +1,158 @@
+package com.example.Huiswerkles10backend.dtos.input;
+
+
+import jakarta.validation.constraints.*;
+
+public class TelevisionInputDto {
```

Comment:
Je hebt op al je inputDto's netjes validatieregels toegepast op je outputDtos niet, heel goed. Alleen je televisionInpuDto heeft private variabelen terwijl je andere inputDtos public variabelen hebben. Het is beter om uniform te zijn